src/storage/btree.c

```diff
@@ -6177,6 +6357,9 @@ void
 btree_scan_clear_key (BTREE_SCAN * btree_scan)
 {
   btree_clear_key_value (&btree_scan->clear_cur_key, &btree_scan->cur_key);
+#if defined(IMPROVE_RANGE_SCAN_IN_BTREE)
+  btree_clear_common_prefix_page_info (btree_scan);
```

Comment:
I think the btree_clear_common_prefix_page_info function is unnecessary.
This is to clear bts->common_prefix_key, but one more step is needed to know this.

Reply:
Are you saying you want to change it to something like a macro?

Reply:
If we do not create a separate struct inside the BTS in the future, it would be better to clear it directly from btree_scan_clear_key.

Are we planning to create a separate struct within BTS in the future?

Reply:
The purpose is to express my opinion, so if you read it, please resolve it.